assembl/static/js/app/views/admin/adminTimelineEvents.js

```diff
@@ -0,0 +1,244 @@
+'use strict';
+/**
+ *
+ * @module app.views.admin.adminTimelineEvents
+ */
+
+var Assembl = require('../../app.js'),
+    Ctx = require('../../common/context.js'),
+    i18n = require('../../utils/i18n.js'),
+    EditableField = require('../reusableDataFields/editableField.js'),
+    TimelineEvent = require('../../models/timeline.js'),
+    LangString = require('../../models/langstring.js'),
+    CollectionManager = require('../../common/collectionManager.js'),
+    Marionette = require('../../shims/marionette.js'),
+    Growl = require('../../utils/growl.js'),
+    SimpleLangStringEditPanel = require('../simpleLangStringEdit.js'),
+    AdminNavigationMenu = require('./adminNavigationMenu.js'),
+    $ = require('jquery'),
+    _ = require('underscore'),
+    Promise = require('bluebird');
+
+
+/**
+ * @class  app.views.admin.adminTimelineEvents.AdminTimelineEventPanel
+ */
+var AdminTimelineEventPanel = Marionette.LayoutView.extend({
+  constructor: function AdminTimelineEventPanel() {
+    Marionette.LayoutView.apply(this, arguments);
+  },
+  template: '#tmpl-adminTimelineEvents',
+
+  ui: {
+    addTimelineEvent: '.js_add_event',
+    save: '.js_save',
+    timelineEventsList: '.js_timelineEventsList',
+    navigationMenuHolder: '.navigation-menu-holder',
+  },
+
+  regions: {
+    timelineEventsList: '@ui.timelineEventsList',
+    navigationMenuHolder: '@ui.navigationMenuHolder',
+  },
+
+  events: {
+    'click @ui.addTimelineEvent': 'addTimelineEvent',
+  },
+
+  initialize: function(options) {
+    var that = this,
+        collectionManager = new CollectionManager();
+    this.timelineEventCollection = null;
+    if(this.isViewDestroyed()) {
+      return;
+    }
+    this.timelinePromise = collectionManager.getAllTimelineEventCollectionPromise().then(function(timeline) {
+      that.timelineEventCollection = timeline;
+      that.render();
+    })
+  },
+
+  addTimelineEvent: function(ev) {
+    var eventCollection = this.timelineEventCollection,
+        lastEventId,
+        event,
+        title = new LangString.Model(),
+        titles = {},
+        description = new LangString.Model(),
+        descriptions = {},
+        preferences = Ctx.getPreferences();
+    if (eventCollection.length > 0) {
+      lastEventId = eventCollection.models[eventCollection.length-1].id;
+    }
+    _.map(preferences.preferred_locales, function(loc) {
+      titles[loc] = '';
+      descriptions[loc] = '';
+    });
+    title.initFromDict(titles);
+    description.initFromDict(titles);
+    event = new TimelineEvent.Model({
+      title: title,
+      description: description,
+      previous_event: lastEventId,
+    });
+    eventCollection.add(event);
+    event.save();
```

Comment:
Your solution works for creation, but I think subsequent edition is easier with my approach; otherwise I have to know how much the frontend model has diverged from the backend model. So I used that approach for everything because I could.